### PR TITLE
doc: virt-install in presence of USB dongle

### PR DESCRIPTION
**Pull request by vlavorini**
## Proposed change

Adding instructions to append the USB dongle to the Virtual Machine

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
The CI is failing, please make sure is passes.

**Comment by mrand** [comment]
Would it be appropriate to include a small warning that USB device numbers can change when using a different USB port, or sometimes even the same USB port with number of USB devices plugged in?   This catches a number of Linux users.

File: source/_includes/installation/operating_system.md [hunk 1]
```diff
@@ -281,6 +281,29 @@ _All these can be extended if your usage calls for more resources._
...
+       Bus 003 Device 004: ID 30c9:0052 Luxvisions Innotech Limited Integrated RGB Camera
+       Bus 003 Device 003: ID 1a86:55d4 QinHeng Electronics SONOFF Zigbee 3.0 USB Dongle Plus V2
+       Bus 003 Device 002: ID 06cb:00fc Synaptics, Inc. 
+       Bus 003 Device 005: ID 8087:0033 Intel Corp. 
+       Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+       Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
+       Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+    ```
+
+    you can recognize the Sonoff dongle at Bus 003 Device 003. So the command to install the VM will become:
```
Comment (frenck) [inline source/_includes/installation/operating_system.md]:
```suggestion
    You can recognize the Sonoff dongle at `Bus 003 Device 003`. So the command to install the VM will become:
```

**Comment by vlavorini** [comment]
Ok for me

**Review by frenck — APPROVED** [review]
Thanks, @vlavorini 👍 

../Frenck

**Pull request merged**